lib/core/logconf.c

```diff
@@ -246,6 +247,7 @@ h2o_logconf_t *h2o_logconf_compile(const char *fmt, int escape, char *errbuf)
                     MAP_EXT_TO_TYPE("process-time", ELEMENT_TYPE_PROCESS_TIME);
                     MAP_EXT_TO_TYPE("response-time", ELEMENT_TYPE_RESPONSE_TIME);
                     MAP_EXT_TO_TYPE("duration", ELEMENT_TYPE_DURATION);
+                    MAP_EXT_TO_TYPE("total", ELEMENT_TYPE_TOTAL);
```

Comment:
The name should be `total-time`. `%{total}x` is too hard to understand. Note also that we have `-time` suffix for other elements.